### PR TITLE
Delete some old GPv1 utils, constants, contracts

### PR DESCRIPTION
**Pull request by anxolin**
Companion PR for https://github.com/cowprotocol/cowswap/pull/550

This PR removes some dependencies to the old GPv1, also removes the contracts and some utils. 

There's further code that can be removed, this PR doesn't aim to clean it all, but aims to remove the dependency and some obviously deprecated code as a first step.

The code was giving me some errors when using absolute imports, so I ended up using relative since we also were using relative anyways most of the times.

### CODE REVIEW

**Comment by alfetopito** [comment]
Since this is removing code, it's potentially backwards incompatible.
I suggest doing a major version bump then

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cowprotocol/cow-js",
-  "version": "0.14.2",
+  "version": "0.15.0",
```
Comment (alfetopito) [inline package.json:3]: Time for a major version bump?

File: src/utils/time.ts [hunk 2]
```diff
@@ -1,19 +0,0 @@
-import { MAX_BATCH_ID } from 'const'
-
-/**
- * Epoch in seconds
- */
-export function getEpoch(): number {
-  return Math.floor(Date.now() / 1000)
-}
```
Comment (alfetopito) [inline src/utils/time.ts:8]: This is still useful/relevant, no?

File: package.json (hunk 1 above)
Comment (anxolin) [reply package.json:3]: Sure :)

File: src/utils/time.ts (hunk 2 above)
Comment (anxolin) [reply src/utils/time.ts:8]: yep, it might, but I don't think we use it any more. I'll leave it just in case

**Review by alfetopito — APPROVED** [review]

**Pull request merged**